src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -16,60 +40,78 @@ const PreviewCard = ({
     display_locale,
     id,
     url,
-    version,
-    created_on,
     concept_class,
     datatype,
     names,
   } = concept;
 
-  const DATE_OPTIONS = {
-    weekday: 'long',
-    year: 'numeric',
-    month: 'short',
-    day: 'numeric',
-    hour: 'numeric',
-    minute: 'numeric',
-  };
-
   const mapping = mappings || 'none';
   const description = descriptions ? descriptions[0].description : 'none';
-  const synonyms = names && names.map(name => name.name).join();
+  const synonyms = names && names
+    .map(name => `<span className='synonym'>${name.name}</span>`)
+    .filter(name => name !== `<span className='synonym'>${display_name}</span>`)
+    .join(' ');
+  let setMembers;
+  let answers;
+  let otherMappings = mapping;
+  let remainingCount;
+  if (mapping !== 'none') {
+    setMembers = getSetMembers(mapping);
+    answers = getAnswers(mapping);
+    otherMappings = getOtherMappings(mapping);
+  }
+
+  const displaySetMembersOrAnswers = () => {
+    if (setMembers && setMembers.length > 0) {
+      remainingCount = setMembers.length - 5;
```

Comment:
What does the magic number 5 stand for?

Reply:
We were required to display five items

Reply:
How would i tell that from the magic number 5?

Reply:
I have fixed it.